signingscript/docker.d/passwords.yml

```diff
@@ -160,6 +160,11 @@ in:
              {"$eval": "AUTOGRAPH_AUTHENTICODE_PASSWORD"},
              ["autograph_authenticode", "autograph_authenticode_stub"]
             ]
+          - ["https://autograph-external.prod.autograph.services.mozaws.net",
+             {"$eval": "AUTOGRAPH_AUTHENTICODE_SHA2_USERNAME"},
+             {"$eval": "AUTOGRAPH_AUTHENTICODE_SHA2_PASSWORD"},
+             ["autograph_authenticode_sha2", "autograph_authenticode_sha2_stub"]
```

Comment:
Not blocking: If I understand correctly, level 1 workers sign the VPN with the production key, right? If so, would you know why?

Reply:
No, they do not.

Reply:
Discussed over Element. While variable names are identical, SOPS provides different values